ichnaea/content/tests/test_views.py

```diff
@@ -87,36 +87,40 @@ def test_stats(self, session, session_tracker, views):
 
 
 class TestFunctionalContent(object):
-    def test_content(self, app, session_tracker, metricsmock):
-        app.get("/", status=200)
-        app.get("/apple-touch-icon-precomposed.png", status=200)
-        app.get("/api", status=200)
-        app.get("/contact", status=200)
-        app.get("/favicon.ico", status=200)
-        app.get("/map", status=200)
-        app.get("/nobody-is-home", status=404)
-        app.get("/optout", status=200)
-        app.get("/privacy", status=200)
-        app.get("/robots.txt", status=200)
-        app.get("/static/css/images/icons-000000@2x.png", status=200)
-        app.get("/terms", status=200)
-        session_tracker(0)
-        app.get("/stats/regions", status=200)
-        session_tracker(1)
-        app.get("/stats", status=200)
-        session_tracker(8)
-        assert metricsmock.has_record(
-            "incr", "request", value=1, tags=["path:", "method:get", "status:200"]
-        )
-        assert metricsmock.has_record(
-            "incr", "request", value=1, tags=["path:map", "method:get", "status:200"]
-        )
-        assert metricsmock.has_record(
-            "timing", "request.timing", tags=["path:", "method:get"]
```

Comment:
``markus.utils.generate_tag("path", "")`` returns "path" instead of "path:". I looked at Grafana, and my guess is the homepage is being registered under ``path=true``. I've changed it to ``path=__homepage__``, which is ugly but maybe self-explanatory? But maybe confused with ``path=__heartbeat__`` and ``path=__version__``. And, typing double underscores in markdown is problematic.

Reply:
Tricky... I think i'd do "home" or "index" without the underscores because the underscores aren't a good signal for anything.

Reply:
I changed to a single period, ``"."``, just before your review. It is gross, I'm not thrilled. I'm chewing on your suggestions, which seem better to me. Also, I'm questioning if request timing should be in metrics at all.

Reply:
I'm changing again to `".homepage"`. The view is called ``homepage_view``, the template is ``homepage.pt``, so that's more consistent wording. None of the other views start with a dash, so hopefully it will stand out in sorting, and is more grep-able, if you remember to escape the period (I use ``ack \\.homepage``). I tried a dash as well (`"-homepage"`), but ack thought I was passing options (`-h`, `-o`, `-m`...).

And now I've thought about this more than anyone else in history.